policy-controller/k8s/status/src/index.rs

```diff
@@ -337,3 +315,24 @@ pub(crate) fn make_patch(
     });
     k8s::Patch::Merge(value)
 }
+
+fn make_accepted_condition(accepted: bool, timestamp: chrono::DateTime<chrono::Utc>) -> Condition {
```

Comment:
nit: I'd maybe call this `to_parent_condition` or something similar? Accepted makes me think that the Condition is itself accepted.

Reply:
We're making the `Accepted` condition here so I do think this is the right name... `Accepted` has a value of true or false and that's what we're passing in.